src/Features/Core/Portable/PullMemberUp/AbstractPullMemberUpRefactoringProvider.cs

```diff
@@ -97,8 +97,8 @@ private bool IsLocationValid(INamedTypeSymbol symbol, Solution solution, Cancell
             foreach (var destination in destinations)
             {
                 var puller = destination.TypeKind == TypeKind.Interface
-                    ? new InterfacePullerWithQuickAction() as AbstractMemberPullerWithQuickAction
-                    : new ClassPullerWithQuickAction();
+                    ? InterfacePullerWithQuickAction.GetInstance as AbstractMemberPullerWithQuickAction
+                    : ClassPullerWithQuickAction.GetInstance;
```

Comment:
super weird to have a field/property caled 'GetInstance'....

Reply:
> super weird to have a field/property caled 'GetInstance'....

I follow @heejaechang 's comments above and change it to singleton. So do you mean I should change it to using static method rather than property?

Reply:
It should be a 'static readonly' field called 'Instance'.  Thanks! :)